client/app/telecom/freeFax/freeFax-main.view.html

```diff
@@ -9,8 +9,9 @@
     </div>
     <!-- /.telecom-section-header -->
 
-    <main class="telecom-section-content"
-          role="main">
+    <main class="telecom-section-content"role="main">
```

Comment:
Plz prodive space or use own attribut row for instance:

```html
<main class="telecom-section-content"
      role="main">
```

Reply:
Changed and I also did a rebase :-)